versioned_docs/version-V3/reference/Deployments.md

```diff
@@ -3,10 +3,53 @@ id: deployments
 title: Deployments
 ---
 
-# Testnet Deploys
+# Uniswap Contract Deployments
 
-The latest version of `@uniswap/v3-core`, `@uniswap/v3-periphery`,
-and supporting contracts (e.g. `WETH9`, `Multicall2`) are
-deployed to testnets.
+The latest version of `@uniswap/v3-core`, `@uniswap/v3-periphery`, and other contracts are deployed at the addresses listed below. They are deployed at the same address on Ethereum mainnet, Optimism, Arbitrum, Polygon, and all testnets.
```

Comment:
Do we want to include v3-staker and swap-router contracts listed out here too? Might be more clear to list what has been deployed based on the repo of the code. (like you did below)